src/router/routes.js

```diff
@@ -87,6 +87,17 @@ const routes = [
         },
         component: () => import('pages/User/Applications.vue')
       },
+      {
+        path: '/privacy',
+        meta: {
+          title: 'Privacy',
+          roles: ['user'],
```

Comment:
For privacy, is accesible by everyone. so you dont need to add roles.

If u add `roles:['user']` then user with role `user` only can access the page.

I tried to access using `admin`.  Then the page is blank.